nixos/modules/security/acme/default.nix

```diff
@@ -566,6 +576,27 @@ let
         example = "/var/src/secrets/example.org-route53-api-token";
       };
 
+      credentials = mkOption {
+        type = types.attrsOf (types.str);
+        inherit (defaultAndText "credentials" {}) default defaultText;
+        description = lib.mdDoc ''
+          Environment variables to set for the cert's service for your selected
+          dnsProvider.
+          To find out what values you need to set, consult the documentation at
+          <https://go-acme.github.io/lego/dns/> for the corresponding dnsProvider.
+          This is an alternative to credentialsFile.
+          Variables suffixed by "_FILE" are passed securely with `LoadCredential=`.
```

Comment:
```suggestion
          Environment variables to set for the cert's service for your selected dnsProvider.
          To find out what values you need to set, consult the documentation at
          <https://go-acme.github.io/lego/dns/> for the corresponding dnsProvider.
          This is an alternative to credentialsFile.
          Variables suffixed by "_FILE" are passed securely with `LoadCredential=`.
```

Reply:
No. This is fine. I don't see the point of this review.

Reply:
It is easier to read and we don't have a hard 80 character line limit

Reply:
I'd really suggest focusing on more substantive review suggestions outside of cases of actively egregious/misleading formatting until there is any kind of agreed consensus on formatting rules (and automated tooling to apply and enforce them). There's no consistency about this in-tree and nitpicking about it with the present state of things just adds busywork for contributors without any real benefit.